cli/sawtooth_cli/block.py

```diff
@@ -134,7 +135,7 @@ def get_block_data(block):
                 print_yaml(block_data)
 
         else:
-            raise CliException('unknown format: {}'.format(args.format))
+            raise AssertionError('unknown format: {}'.format(args.format))
```

Comment:
This actually is a `CliException`,  since we're not enforcing it via argparse.  You can either add `choices=['json', 'yaml', 'csv']` to the `--format` definition, and keep the `AssertionError`, or change this back to a `CliException`.